docs/LDES.md

```diff
@@ -0,0 +1,111 @@
+
+We should also **watch** the LDES feed from GN for instances of Besluit with the right type (the one that makes a mandataris official) that are linked to a Mandataris (which on our side will be in draft). If we find such a Besluit, we should switch the Mandataris to official and create that link on our end too.
+
+However, there is no guarantee that the besluit will be created in GN. Therefore it should be possible to mark a mandataris as official (as opposed to draft) manually, even if no event was received from GN. In that case, a link to the meeting notes of the corresponding meeting can be provided manually as well.
+
+### Publishing to Vlaamse Mandatendatabank
+
+LMB extracts the Mandaten and Leidinggevenden scope from Loket. Therefore, it should also pick up the role of sharing this information with the central publication system, specifically the Vlaamse Mandaten Databank (VMDB). This will also be done through an LDES stream.
+
+Since suppliers of the meeting note software (filling the role of Gelinkt Notuleren) understand SPARQL and are currently using a SPARQL endpoint on Loket for syncing, we should use the same principle for the draft Mandataris instances. This means that we expect the Vlaamse Mandatendatabank to also listen to our Draft Mandataris LDES feed and internalize this information into their SPARQL endpoint OR that we listen to the local LDES feeds in the central Loket version of LMB and provide the sparql endpoint there.
+
+## Types exposed on LDES feed
+
+VMDB will be interested in instances of the following types:
+
+- **mandaat:Mandataris:** but only the Mandataris instances in the official state
+- **mandaat:Fractie:** as the local government can define their own instances of Fracties to link a Mandataris to
+- **org:Membership:** when used to define the membership of a Mandataris of a Fractie
+- **ext:BeleidsdomeinCode:** as a mandataris can be linked to these concepts and a local government can define their own set of concepts here
+- **person:Person:** normally, the Person a Mandataris links to can be found in the election results and so this information should be globally available (and non-modifiable). However, there are cases where an unelected person can still be called in to take a Decretaal mandaat. These should be published too so the Person a Mandataris links to has meaning. This also happens for the head of OCMW organisation, who can be made schepen. Luckily, there is an api to get the uri of a person given you have the RRN. Person still needs to be on the LDES, but this likely means we need to secure our LDES as there can be no personal info on public LDES feed
```

Comment:
Few marks about this paragraph:
- you say a person can be called in to take a Decretaal mandaat, I think you don't really say "take a mandate", better would be "assume a mandate"
- a bit further "these should be published too" is written, too is not correct here I think, "these are also published" would be better
- the secure at the end is not really clear, is this about encryption, or more about data breaches?